### PR TITLE
Added Democracy pallet

### LINKED ISSUE #152: [Trappist on Rococo] Add democracy pallet to Rococo.
If we want to replicate a more real production experience when managing the live version of Trappist, we need to configure the democracy pallet like for example executing runtime upgrades.

<Specs and more details are still to be discussed and defined>

### PR DESCRIPTION
**Pull request by valentinfernandez1**
This PR includes the democracy pallet into the Trappist runtime as part of the  issue #152.

Most value are based of the Rococo config for this pallet. The Periods can be discussed depending on the objective of the project and how is going to be intended to be used by other teams.

```rust
type Slash = (); 
```
Slash was defined empty for the same reason. Once we define how different teams are intended to interact with the pallet this can be modified.

Closes #152

### CODE REVIEW

File: runtime/trappist/src/lib.rs
Comment (hbulgarini) [inline runtime/trappist/src/lib.rs:28]: Minor. For consistency i would import `DAYS` from here:
https://github.com/paritytech/trappist/pull/159/files#diff-1f7754547bbff248c6d0c46b32d579f4ecd082d31b014b6d141d116d6f2e0df0R64

File: runtime/trappist/src/lib.rs
Comment (hbulgarini) [inline runtime/trappist/src/lib.rs:498]: What needs to be checked here?

File: runtime/trappist/src/lib.rs
Comment (hbulgarini) [inline runtime/trappist/src/lib.rs:507]: This can be kept empty until we add the Treasury pallet and we point the slashes to there.

**Review by hbulgarini — COMMENTED** [review]
I have left some comments & questions.

I would wait for this PR to be merged until #158 is merged though.

File: runtime/trappist/src/lib.rs
Comment (valentinfernandez1) [reply runtime/trappist/src/lib.rs:498]: My bad i forgot to delete that comment, I was getting an error with rust-analyzer

**Review by stiiifff — APPROVED** [review]

**Pull request merged**
